### PR TITLE
Fix/web 41

### PR DESCRIPTION
**Pull request by Jmorales1306**
## Description
-
1)Borrado de @types pertenecientes a tsen los diferentes package.json  
2)Agregado de variante de primary-color
3)Creacion de un nuevo package, encargado de manejar context compartidos entre web y los componente
4)Eliminacion de estilos repetidos
5)Mejora de la transicion en el submenu de idioma

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [X] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed. (Add screenshot)
- [ ] Is my code avoiding performance problems (ex: rendering issues, amount of queries against the database, n+1 problem, infinite loops, memory usage)
- [ ] Is my code handling integration exceptions (ex: timeout error while calling an external service)
- [ ] My code includes usage metrics (ex: google analytics events, logs)
- [ ] My code includes restriction of access based on roles and permissions

## Pull request type
 
 

Please check the type of change your PR introduces:
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

## JIRA Ticket Link

JIRA Ticket:
WEB-41

## Commands to be executed or config files to be changed
-

### CODE REVIEW

File: packages/components/src/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -102,31 +98,31 @@ const Navbar = ({ navItems }) => {
 
   return (
     <nav
-      className={`${isSticky ? 'sticky navBg z-10' : 'absolute bg-transparent'} w-full top-0 start-0 ease-out duration-500 `}
+      className={`${isSticky ? 'sticky navBg z-10' : 'absolute'} w-full top-0 start-0 ease-out duration-500 `}
     >
       {/* navbar desktop */}
       <div
-        className={` max-w-[1553px] flex flex-wrap items-center justify-between  p-4 bg-white md:bg-white lg:bg-white md:mx-auto lg:mx-auto xl:mx-auto 2xl:mx-auto`}
+        className={`max-w-[1553px] flex flex-wrap bg-white lg:bg-transparent items-center justify-between p-4`}
```
Comment (HamiltonG98) [inline packages/components/src/Navbar/Navbar.jsx:105]: Mantener el orden de las clases

File: packages/components/src/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -102,31 +98,31 @@ const Navbar = ({ navItems }) => {
...
         </a>
 
         {/* Desktop Navigation */}
-        <ul className="hidden xl:flex 2xl:flex z-10">
+        <ul className="hidden lg:flex z-10">
           {navItems.map((item) => (
             <li
               key={item.id}
-              className={`leading-5 ${isSticky ? 'font-normal' : 'font-medium'} md:ml-6`}
+              className={`lg:ml-4 leading-5 ${isSticky ? 'font-normal' : 'font-medium'}`}
```
Comment (HamiltonG98) [inline packages/components/src/Navbar/Navbar.jsx:117]: Mantener el orden de las clases

**Review by HamiltonG98 — COMMENTED** [review]
En general revisa el orden de las clases que tenes en todos los componentes de este PR

File: packages/components/src/Navbar/Navbar.jsx (hunk 2 above)
Comment (Jmorales1306) [reply packages/components/src/Navbar/Navbar.jsx:117]: En este, no tiene mas peso la condicional de estilo que cambia en funciona al estado ?

Comment (HamiltonG98) [reply packages/components/src/Navbar/Navbar.jsx]: Si, pero tenes un lg de primero y despues un leading que es el que deberia estar de primero

Comment (Jmorales1306) [reply packages/components/src/Navbar/Navbar.jsx]: Entendio, ya lo resolvi en las ultimas correcciones eso.

**Review by HamiltonG98 — APPROVED** [review]

**Pull request merged**